### ISSUE TITLE
The accuracy rate was only 0.5

### ISSUE THREAD

**Issue by quantum-soul** [brijeshiitg/Pytorch-implementation-of-SRNet#4]
I tried to train the model with 0.4bpp suniward, but loss did not decrease all the time. The accuracy rate was only 0.5. Could you please provide the training code? Thank you

**Comment by zzb112233** [comment]
> I tried to train the model with 0.4bpp suniward, but loss did not decrease all the time. The accuracy rate was only 0.5. Could you please provide the training code? Thank you

Hi,friend,have you solved the problem? Mine is 0.5 too,I`m wondering maybe the computation of loss is mismatch.

**Comment by eardroppers** [comment]
same as 0.5 here.. but i am using keras ..strange...

**Comment by brijeshiitg** [comment]
Use data augmentation.

**Comment by lzhamb** [comment]
Hello! I also encountered this problem when running this code. Could you please guide me on how to specifically perform data augmentation?
哈喽前辈，我在运行这个代码的时候也出现了这个问题，您能指导我一下具体要怎么进行数据增强吗?

**Comment by eardroppers** [comment]
同学您好，

目前就是两种模式，一种就是自己根据架构重新写SRNET的代码，另外一种就是对比的时候不使用SRNET，因为拟合非常困难。




你可以使用siastegnet,zhunet 进行对比实验。







-----原始邮件-----
发件人:lzhamb ***@***.***>
发送时间:2025-11-28 00:03:42 (星期五)

收件人: brijeshiitg/Pytorch-implementation-of-SRNet ***@***.***>
抄送: eardroppers ***@***.***>, Comment ***@***.***>
主题: Re: [brijeshiitg/Pytorch-implementation-of-SRNet] The accuracy rate was only 0.5 (#4)



lzhamb left a comment (brijeshiitg/Pytorch-implementation-of-SRNet#4)

Hello! I also encountered this problem when running this code. Could you please guide me on how to specifically perform data augmentation?
哈喽前辈，我在运行这个代码的时候也出现了这个问题，您能指导我一下具体要怎么进行数据增强吗?

**Comment by lzhamb** [comment]
> 同学您好，
> 
> 目前就是两种模式，一种就是自己根据架构重新写SRNET的代码，另外一种就是对比的时候不使用SRNET，因为拟合非常困难。
> 
> 
> 
> 
> 你可以使用siastegnet,zhunet 进行对比实验。
> […](#)

感谢前辈的指导🥰
